docs/proposals/apb-state-support.md

```diff
@@ -0,0 +1,59 @@
+# Abstract
+
+This proposal will outline a simple and secure method for allowing Service Bundle developers to set state during a Service Bundle
+action and then be able to access that state during a subsequent Service Bundle action.
+
+## Actions
+   - provision
+   - deprovision
+   - update
+   - bind 
+   - unbind
+   
+   
+## Problem Description
+
+Service Bundle's are stateless. All state is managed for them by the broker. Currently the broker passes in parameters specified
+during a request to the catalog. It also passes in some additional parameters such as the namespace etc, additionally 
+we also pass in credentials created during the provision. While this is useful, there is no mechanism for a Service Bundle to store 
+and access data across actions without exposing it to the end user. We want to avoid Service Bundle developers working around this 
```

Comment:
> We want to avoid Service Bundle developers working around this

Big +1, if there's enough desire to have folks hacking around to make this happen, we need to provide first-class tools to enable them.